app/main.py

```diff
@@ -16,4 +16,32 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {}
```

Comment:
Using a global variable `sorted_variables` to store results can lead to unintended side effects if the function is called multiple times. Consider returning a new dictionary from the function instead of modifying a global variable.